app/src/main/java/com/example/lastfm/ArtistInfoFragment.java

```diff
@@ -36,15 +38,17 @@
     private static final int PICTURE_SIZE = 3;
 
     @Override
-    protected void onCreate(Bundle savedInstanceState) {
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                                Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_artist_screen);
-        tvNameView = findViewById(R.id.artist_info_name);
-        tvPlayCount = findViewById(R.id.artist_info_playcount);
-        artistImage = findViewById(R.id.big_artist_image);
-        tvArtistBio = findViewById(R.id.artist_info_bio);
-        Intent intent = getIntent();
-        String name = intent.getStringExtra("artistName");
+        View view = inflater.inflate(R.layout.artist_info_fragment, container, false);
+        tvNameView = view.findViewById(R.id.artist_info_name);
+        tvPlayCount = view.findViewById(R.id.artist_info_playcount);
+        artistImage = view.findViewById(R.id.big_artist_image);
+        tvArtistBio = view.findViewById(R.id.artist_info_bio);
+        Bundle bundle = this.getArguments();
```

Comment:
u can execute this function without this

Reply:
Changed